modules/navier_stokes/src/fv/kernels/INSFVGravityForce.C

```diff
@@ -0,0 +1,47 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "INSFVGravityForce.h"
+
+registerMooseObject("NavierStokesApp", INSFVGravityForce);
+
+InputParameters
+INSFVGravityForce::validParams()
+{
+  InputParameters params = FVElementalKernel::validParams();
+  params.addClassDescription("Computes a body force due to gravity.");
+  params.addRequiredParam<RealVectorValue>("gravity", "Direction of the gravity vector");
+  MooseEnum momentum_component("x=0 y=1 z=2", "x");
```

Comment:
```suggestion
  MooseEnum momentum_component("x=0 y=1 z=2");
```
No need for a default if it's required?

Reply:
:+1: